### DISCUSSION TITLE
Why is pending_index_rows stuck at a non zero number?

### DISCUSSION THREAD

**Discussion by LiuDaveLiu** [milvus-io/milvus#30804] [Q&A and General discussion]
See snippet below, any thing to be concerned about?
![image](https://github.com/milvus-io/milvus/assets/8761973/4ca4f26e-0cd8-4fb0-bade-62b9183c3eee)

**Comment by xiaofan-luan** [comment]
is there any new insertion rows?

Indexed rows should match total_rows, but that only indicate by the time you start build index, how many rows are there wait for index build.

I didn't take a deep look into the code, but my rough guess is "pending index rows" are the actual rows waiting to build index.  It could be always a non zero value unless you stop inserting
